.circleci/config.yml

```diff
@@ -633,11 +710,536 @@ jobs:
+      - *restore_cache
+      - *add_ssh_keys
+      - *prepare_environment
+      - run:
+          name: Wait until server ready
+          shell: /bin/bash
+          when: always
+          command: |
+            python3 ./Tests/scripts/wait_until_server_ready.py "Demisto two before GA"
+      - run:
+          name: Run Tests - Server 4.1
+          shell: /bin/bash
+          when: always
+          command: |
+            echo 'export INSTANCE_ROLE="Demisto two before GA"' >> $BASH_ENV
+            source $BASH_ENV
+
+            export TEMP=$(cat ./Tests/filter_envs.json | jq '."Demisto two before GA"')
+            echo "Demisto two before GA filter=$TEMP"
+            if [[ "$TEMP" == "false" ]];
+              then
+                echo "Skipping - instance was not setup"
+                exit 0
+            fi
+            if ./Tests/scripts/is_ami.sh ;
+              then
+                ./Tests/scripts/run_tests.sh "Demisto two before GA"
+
+              else
+                echo "Not AMI run, can't run on this version"
+            fi
+      - *destroy_instances
+      - *store_artifacts
+  Server 4_5:
+    <<: *container_config
+    resource_class: medium+
+    <<: *environment
+    steps:
+      - setup_remote_docker
+      - *attach_workspace
+      - *add_ssh_keys
+      - *prepare_environment
+      - run:
+          name: Wait until server ready
+          shell: /bin/bash
+          when: always
+          command: |
+            python3 ./Tests/scripts/wait_until_server_ready.py "Demisto one before GA"
+      - run:
+          name: Run Tests - Server 4.5
+          shell: /bin/bash
+          when: always
+          command: |
+            echo 'export INSTANCE_ROLE="Demisto one before GA"' >> $BASH_ENV
+            source $BASH_ENV
+            export TEMP=$(cat ./Tests/filter_envs.json | jq '."Demisto one before GA"')
+            echo "Demisto one before GA filter=$TEMP"
+            if [[ "$TEMP" == "false" ]];
+              then
+                echo "Skipping - instance was not setup"
+                exit 0
+            fi
+            if ./Tests/scripts/is_ami.sh ;
+              then
+                ./Tests/scripts/run_tests.sh "Demisto one before GA"
+
+              else
+                echo "Not AMI run, can't run on this version"
+            fi
+      - *destroy_instances
+      - *store_artifacts
+  Server 5_0:
+    <<: *container_config
+    resource_class: medium+
+    <<: *environment
+    steps:
+      - *attach_workspace
+      - setup_remote_docker
+      - *restore_cache
+      - *add_ssh_keys
+      - *prepare_environment
+      - run:
+          name: Wait until server ready
+          shell: /bin/bash
+          when: always
+          command: |
+            if ./Tests/scripts/is_ami.sh ;
+              then
+                python3 ./Tests/scripts/wait_until_server_ready.py "Demisto GA"
+              else
+                python3 ./Tests/scripts/wait_until_server_ready.py "Demisto"
```

Comment:
```suggestion
                python3 ./Tests/scripts/wait_until_server_ready.py "master"
```

Reply:
Actually this part is on purpose

This script unlike run tests looks at the file `Tests/instance_ips.txt` where it has lines with `role`:`ip`

For some reason when using non-ami builds the role name is `Demisto` even though in `env/results.json` it is master